### PR TITLE
ICMSLST-360 Move Derogation form help texts and labels to models

### PR DESCRIPTION
**Pull request by kevinetienne**
* use `application_pk` as parent path
* fix submitting application without a supporting document (ICMSLST-731)
* makes `orgin_country` and `consignment_country` non blank to forced them to be required by default

### CODE REVIEW

File: web/domains/case/_import/derogations/models.py [hunk 1]
```diff
@@ -1,22 +1,55 @@
...
+        verbose_name="Commodity Code",
+        help_text=(
+            "It is the responsibility of the applicant to ensure that the commodity code in"
+            " this box is correct. If you are unsure of the correct commodity code,"
+            " consult the HM Revenue and Customs Integrated Tariff Book, Volume 2,"
+            " which is available from the Stationery Office. If you are still in doubt,"
+            " contact the Classification Advisory Service on (01702) 366077."
+        ),
+    )
+    goods_description = models.CharField(
```
Comment (oskusalerma) [inline web/domains/case/_import/derogations/models.py:36]: can you separate multi-line model fields with empty lines

File: web/domains/case/_import/derogations/views.py [hunk 2]
```diff
@@ -65,10 +70,11 @@ def edit_derogations(request: AuthenticatedHttpRequest, *, application_pk: int)
 
 @login_required
 @permission_required("web.importer_access", raise_exception=True)
-def add_supporting_document(request: AuthenticatedHttpRequest, pk: int) -> HttpResponse:
-
+def add_supporting_document(request: AuthenticatedHttpRequest, application_pk: int) -> HttpResponse:
     with transaction.atomic():
-        application = get_object_or_404(DerogationsApplication.objects.select_for_update(), pk=pk)
+        application = get_object_or_404(
```
Comment (oskusalerma) [inline web/domains/case/_import/derogations/views.py:75]: typehint this?

**Review by oskusalerma — CHANGES_REQUESTED** [review]

File: web/domains/case/_import/derogations/views.py (hunk 2 above)
Comment (kevinetienne) [reply web/domains/case/_import/derogations/views.py]: Updated in https://github.com/uktrade/icms/pull/452/commits/b4b1d525ee9a5cd2e1cb56206fa4a0f48ee53f31

File: web/domains/case/_import/derogations/models.py (hunk 1 above)
Comment (kevinetienne) [reply web/domains/case/_import/derogations/models.py:38]: Updated in https://github.com/uktrade/icms/pull/452/commits/b4b1d525ee9a5cd2e1cb56206fa4a0f48ee53f31

**Review by oskusalerma — APPROVED** [review]

**Pull request merged**